Library/ViewModels/RewardsCollectionViewModel.swift

```diff
@@ -177,9 +177,28 @@ public final class RewardsCollectionViewModel: RewardsCollectionViewModelType,
     // Tracking
     Signal.combineLatest(project, selectedRewardFromId, refTag)
       .observeValues { project, reward, refTag in
+
+        // The `Backing` is nil for a new pledge.
+        let backing = project.personalization.backing
```

Comment:
Do we have tests for this logic and the checkoutPropertiesData? It seems to be critical logic for these properties. Do we also use this anywhere else?

Reply:
Good call. Added some tests to handle whether there is a Backing or not